media-gfx/gnofract4d/gnofract4d-4.3_p20201029.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8,9} )
+DISTUTILS_SINGLE_IMPL=1
+
+inherit distutils-r1 optfeature virtualx xdg
+
+COMMIT="3e3893b0ee381098418d7b28997f6b861c53eff1"
+
+DESCRIPTION="A program for drawing beautiful mathematically-based images known as fractals"
+HOMEPAGE="https://fract4d.github.io/gnofract4d/"
+SRC_URI="https://github.com/fract4d/gnofract4d/archive/${COMMIT}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+COMMON_DEPEND="
+	media-libs/libpng:0=
+	virtual/jpeg"
+DEPEND="${COMMON_DEPEND}
+	test? (
+		media-video/ffmpeg[vpx,zlib]
+	)"
```

Comment:
Please put the `test? ( )` block into BDEPEND. It then allows you to get rid of `COMMON_DEPEND`.
https://dev.gentoo.org/~mgorny/python-guide/distutils.html#customizing-the-test-phase

Reply:
OK, that works - it is only the ffmpeg executable used during testing. Updated as suggested.